containers/podman-desktop
# feat: add compatibility enable / disable for Windows

feat: add compatibility enable / disable for Windows

### What does this PR do?

* Adds functionality to the pre-existing "Docker Compatibility" button to
enable / disable Docker Compatibility on Windows
* Fixes tests from previous #1697 
* Implements "restart podman machine" function for future button for #1697 
* Small changes to extension.ts functionality (showing the information).

### Screenshot/screencast of this PR



![Screenshot 2023-03-22 at 11 32 38 AM](https://user-images.githubusercontent.com/6422176/227009178-dfac4a49-edf9-4c18-b0b5-9dd08f00f88c.png)


### What issues does this PR fix or reference?



Closes part of https://github.com/containers/podman-desktop/issues/1447

### How to test this PR?



On Windows, click on the "Docker Compatibility" button and follow the
steps to enable / disable. This should be user friendly. Any comments please mention!

Signed-off-by: Charlie Drage <charlie@charliedrage.com>
